lib/util/support.js

```diff
@@ -1,7 +1,7 @@
-/* globals chrome, navigator, RTCPeerConnection */
+/* globals chrome, navigator */
 'use strict';
 
-const { guessBrowser } = require('@twilio/webrtc/lib/util');
+const { guessBrowser, support: isWebRTCSupported } = require('@twilio/webrtc/lib/util');
```

Comment:
isnt `support` a very generic name? I suggest rename it to `isWebRTCSupported`?

Reply:
@makarandp0 Since it is a util method within the WebRTC lib, I think its fine. That is why I'm renaming it when I'm consuming the method in twilio-video.js. I'm following the same naming convension as the `support()` function in twilio-video.js.

Reply:
@mmalavalli - Do you mean twilio-video.isSupported property? I think that would be fine too or even better isWebRTCSupported, but export name `support()` sounds very confusing to me.

Reply:
@makarandp0 I think `TwilioWebRTC.isWebRTCSupported` is redundant.

Reply:
@mmalavalli - I think the difference is twilio-webrtc is a public module, and its exports are not internal (unlike `support` function that is completely internal to twilio-video.js.
But Generally I think you are better at naming than me and if you really think it needs to be twilio-webrtc.support,   Lets go for it.